.github/workflows/pages.yml

```
@@ -0,0 +1,44 @@
+---
+name: Deploy GH Pages
+
+on:
+  workflow_dispatch:
+
+jobs:
+  deploy-pages:
+    runs-on: ubuntu-latest
```

```suggestion
    if: github.repository == 'RedHat-Israel/red-hat-israel-site'
    runs-on: ubuntu-latest
```

this would prevent deploys from forks